### PR TITLE
[KOGITO-1884] - Clean up old cucumber test namespaces

### PR DESCRIPTION
**Pull request by msmagnanijr**
See: https://issues.redhat.com/browse/KOGITO-1884

Signed-off-by: Mauricio Magnani <mmagnani@redhat.com>

### CODE REVIEW

**Comment by msmagnanijr** [comment]
sorry guys, I closed the other PR unintentionally. Anyway I tested this script and it is working as it should. In the other PR @radtriste  said something about creating a Jenkinsfile to do that. Do you think this is better? I thought about leaving it on a cron and running it every night. What do you think?

**Comment by radtriste** [comment]
> I thought about leaving it on a cron and running it every night. What do you think?

What do you mean ?
It should be in a Jenkins job so we know the status. You can then simply write the cron jobs in a Jenkinsfile.
This gives more visibility to everybody :)

File: tools/clean-old-namespaces.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+IN=$(kubectl get namespace -o jsonpath='{range .items[*].metadata}{.name}{";"}{.creationTimestamp}{"\n"}{end}' | awk '/cucumber-/')
+projects=$(echo $IN | tr "\n" "\n")
+date_cleanup=$(date -u +"%Y-%m-%dT%H:%M:%SZ" -d "1 day ago")
+for values in $projects
+do
+    name_project=${values%;*}
+    date_project=${values#*;}
+    if [[ "$date_project" < "$date_cleanup"  ]] ; then
+	oc delete project $name_project
```
Comment (radtriste) [inline tools/clean-old-namespaces.sh:11]: Can you add an `echo "delete project ${name_project}"` for visibility on what the script does in logs ?

**Review by radtriste — COMMENTED** [review]
tested and looks good, just one thing

File: tools/clean-old-namespaces.sh (hunk 1 above)
Comment (msmagnanijr) [reply tools/clean-old-namespaces.sh:11]: done

**Comment by msmagnanijr** [comment]
> > I thought about leaving it on a cron and running it every night. What do you think?
> 
> What do you mean ?
> It should be in a Jenkins job so we know the status. You can then simply write the cron jobs in a Jenkinsfile.
> This gives more visibility to everybody :)

Ahh yes, it makes more sense.

**Review by radtriste — COMMENTED** [review]
lgtm

**Review by spolti — APPROVED** [review]

**Pull request merged**